<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diag_log_detail</class>
 <widget class="QDialog" name="diag_log_detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Details</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Project:</string>
     </property>
     <property name="buddy">
      <cstring>line_project</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="line_project">
     <property name="styleSheet">
      <string notr="true">background-color:palette(Window);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Timestamp:</string>
     </property>
     <property name="buddy">
      <cstring>line_timestamp</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="line_timestamp">
     <property name="styleSheet">
      <string notr="true">background-color:palette(Window);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Summary:</string>
     </property>
     <property name="buddy">
      <cstring>line_summary</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="line_summary">
     <property name="styleSheet">
      <string notr="true">background-color:palette(Window);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Detail:</string>
     </property>
     <property name="buddy">
      <cstring>text_detail</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="text_detail">
     <property name="styleSheet">
      <string notr="true">background-color:palette(Window);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>diag_log_detail</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>diag_log_detail</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
